pkg/util/validation/limits.go

```diff
@@ -62,6 +62,11 @@ type Limits struct {
 	CardinalityLimit    int           `yaml:"cardinality_limit"`
 	MaxCacheFreshness   time.Duration `yaml:"max_cache_freshness"`
 
+	// Ruler defaults and limits.
+	// Delay the evaluation of all rules by a set interval to give a buffer
```

Comment:
I think this comment belongs in the ruler itself right before we lookup the tenant limit.